lib/http/browser.js

```diff
@@ -13,6 +14,7 @@ const Base = require('./base');
  * @constructor
  */
 function BrowserHttpClient(connectionConfig) {
+  Logger.getInstance().trace('Initializing BrowserHttpClient with connection config');
```

Comment:
Is it worth adding more details from connectionConfig?

Reply:
Thanks, I added them for all HttpClients with new describeIdentityAttributes() method of ConnectionConfig objects. 

Important note tho - I believe it is not possible to unambiguously identify the connection for which the HttpClient / ConnectionConfig is used, by looking only at those attributes - they can be reused for multiple connections. Nevertheless they increase the traceability of events represented in logs - what costs us quite increased verbosity in those situations.

Reply:
Yes, you are right, @sfc-gh-fpawlowski. However, we can not log all connection config data.